README.md

```diff
@@ -148,6 +148,10 @@ Yes. Please use the following alternatives:
 -	type=GlobalReference, local=False: a concept description object is not defined, just a reference to an external source is made
 (Answered: 2020-09-21)
 
+**[Is it allowed to have Submodel Templates and Submodel Instances in the same AAS?](#id42)** <a name="id42"></a> 
```

Comment:
Please change to:
Is it allowed to have both, Submodel Templates and Submodel Instances, in the same AAS?